libs/domain/order/entries/src/entries.component.ts

```diff
@@ -17,7 +17,10 @@ import { styles } from './entries.styles';
 @defaultOptions({
   limit: 5,
   threshold: 3,
-})
+  enableItemImage: true,
+  enableItemId: true,
+  enableItemPrice: false,
+} as OrderEntriesOptions)
```

Comment:
Typecasting

Reply:
but very useful as long as we refactor; keeping it until we have proper typing in place.